kroki/plugin.py

```diff
@@ -92,7 +92,7 @@ def _save_kroki_image_and_get_url(self, file_name, image_data, files):
         mkdocs_file = File(get_url, self._tmp_dir.name, self._output_dir, False)
         files.append(mkdocs_file)
 
-        return f'/{get_url}'
+        return {mkdocs_file.url_relative_to(page.file)}
```

Comment:
Shouldn't this still keep get_url ? Like, if I have more than one diagram on my page ?